### ISSUE TITLE
"Evolution" in READCOL messages

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#1014]
in gdl-1.0.0rc3-git210517 the behavior of warnings in READCOL (provided by idlastro) was OK
in gdl-1.0.0rc3-git210524 the warnings are now too verbose

old good
```
GDL> path_add, '~/IDLetGDL/Librairies/Git210715/idlastro/pro/
% Compiled module: PATH_ADD.
No explicit position : the defaut is AT THE BEGINNING
GDL> readcol, '~/JPB/ISRF.DAT', a,b
% Compiled module: READCOL.
% READCOL: Format keyword not supplied - All columns assumed floating point
% Compiled module: GETTOK.
% Compiled module: STRNUMBER.
% READCOL: Skipping Line (check_numeric) 1
% READCOL: Skipping Line (check_numeric) 2
% READCOL: Skipping Line (check_numeric) 3
% READCOL: Skipping Line (check_numeric) 4
% READCOL: Skipping Line (check_numeric) 5
% READCOL: Skipping Line (check_numeric) 6
% READCOL: Skipping Line (n_elements) 7
% READCOL: 205 valid lines read
```

current "bad" : 
```
GDL> readcol, '~/JPB/ISRF.DAT', a,b
% Compiled module: READCOL.
% READCOL: Format keyword not supplied - All columns assumed floating point
% Compiled module: GETTOK.
% Compiled module: STRNUMBER.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'DUSTEM:' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'DUSTEM:' to FLOAT.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'Mathis' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'Mathis' to FLOAT.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'Blackbody' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'Blackbody' to FLOAT.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'dilution' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'dilution' to FLOAT.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'Nbr' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'Nbr' to FLOAT.
% Type conversion error: Unable to convert given STRING: '#' to DOUBLE.
% Type conversion error: Unable to convert given STRING: '#' to FLOAT.
% Type conversion error: Unable to convert given STRING: 'wave' to DOUBLE.
% Type conversion error: Unable to convert given STRING: 'wave' to FLOAT.
% READCOL: Skipping Line (n_elements) 7
% READCOL: 211 valid lines read
```

**Comment by GillesDuvert** [comment]
moreover, the result (a and b) is not ok, which is really bad.

**Comment by GillesDuvert** [comment]
closed by #1015
